source/includes/administration/_discounts.md

```diff
@@ -142,3 +200,76 @@ Optional | &nbsp;
 ------- | -----------
 `cutoffDate`<br/>*date* | The date on which the discount will no longer be available to customers who have not already received it. If not provided, the discount will always be available after the start date. 
 `durationMonths`<br/>*integer* | Duration of the discount once it has been applied to a customer. If not provided the discount will last indefinitely, or until credit values are reached.
+
+<!-------------------- EDIT DISCOUNT -------------------->
+### Edit discount
+
+`PUT /applied_pricings/:applied_pricing_id/discounts/:id`
+
+Edit an existing discount that hasn't ended. Only the name and cutoff date can be edited for current discount. All fields can be edited for upcoming discounts.
+
+```shell
+# Edit an existing discount
+curl -X PUT "https://cloudmc_endpoint/rest/applied_pricings/efd32752-c6f2-45cf-b494-cc6be8a45845/discounts/08695c90-3e41-4be8-a3a3-df964620ba62" \
+   -H "MC-Api-Key: your_api_key"
+```
+> Request body example:
+
+```json
+{
+  "name": {
+    "en": "End of summer Discount",
+    "fr": "Réduction estival fin d'été"
+  },
+  "durationMonths": 4,
+  "discountedProducts": {},
+  "discountedCategories": {
+    "8cf73cc0-b86e-49b4-a102-6102894f7955": 2, 
+    "00358632-5c9a-4164-a9a9-df271a9c06a9": 22
+  },
+  "discountScope": "CATEGORIES", 
+  "cutoffDate": "2021-08-24T00:00:00.000Z",
+  "applyToNewCustomersOnly": false,
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+      "applyToNewCustomersOnly": false,
+      "discountedProducts": {},
+      "durationMonths": 4,
+      "type": "PERCENTAGE",
+      "discountScope": "CATEGORIES",
+      "discountedCategories": {
+        "8cf73cc0-b86e-49b4-a102-6102894f7955": 2, 
+        "00358632-5c9a-4164-a9a9-df271a9c06a9": 22
+      },  
+      "name": {
+        "en": "End of summer Discount",
+        "fr": "Réduction estival fin d'été"
+      },
+      "id": "18db7bc6-8be1-48bb-bab1-77a7d696fa3b",
+      "appliedPricing": {
+        "id": "efd32752-c6f2-45cf-b494-cc6be8a45845"
+      },
+      "startDate": "2021-07-23T00:00:00.000Z",
+      "cutoffDate": "2021-08-24T00:00:00.000Z",
+      "status": "UPCOMING"
+  }
+}
+```
+
+Optional | &nbsp;
```

Comment:
these are all optional right? If I dont provide a name in the api call will it just use the existing one or throw an error?

Reply:
just use the existing one.